css/cssom-view/scroll-behavior-element.html

```diff
@@ -186,4 +186,20 @@
       assert_equals(scrollingElement.scrollTop, 0, "Final value of scrollTop");
     });
   }, "Aborting an ongoing smooth scrolling on an element with an instant scrolling");
+
+  promise_test(() => {
+    scrollNode(scrollingElement, "scroll", "smooth", elementToRevealLeft, elementToRevealTop);
+    return waitForScrollEnd(scrollingElement).then(() => {
+      assert_equals(scrollingElement.scrollLeft, elementToRevealLeft, "Final value of scrollLeft after first smooth scroll");
+      assert_equals(scrollingElement.scrollTop, elementToRevealTop, "Final value of scrollTop after first smooth scroll");
+      scrollNode(scrollingElement, "scroll", "instant", 0, 0);
+      assert_equals(scrollingElement.scrollLeft, 0);
+      assert_equals(scrollingElement.scrollTop, 0);
```

Comment:
you can probably test nonzero values, to get something different than resetScroll